INSTALL.md

```diff
@@ -270,13 +270,19 @@ If this works, then proceed to [Installation](#installation). Otherwise, ask for
 
 >   This guide installs libpng via Homebrew as it is the easiest method, however advanced users can install libpng through other means if they so desire.
 </details>
+<details>
+    <summary><i><strong>Note for Apple Silicon (M1) Mac users...</strong></i></summary>
+
+>   Currently, Homebrew and libng must be installed via Rosetta on Apple Silicon Macs. Before continuing, create a [Terminal shell profile with Rosetta](https://www.astroworldcreations.com/blog/apple-silicon-and-legacy-command-line-software). Be sure to run the commands corresponding to Apple Silicon (M1).
+</details>
 
 1. Open the Terminal.
 2. If Homebrew is not installed, then install [Homebrew](https://brew.sh/) by following the instructions on the website.
 3. Run the following command to install libpng.
 
     ```bash
-    brew install libpng
+    brew install libpng # Intel Macs
+    /usr/local/bin/brew install libpng # Apple Silicon (M1) Macs
```

Comment:
Imo these changes are not necessary

Reply:
I still can't speak to M1, but now on an M3 I can add that in addition to `brew install libpng` I needed to install pkg-config with `brew install pkg-config` in order for libpng to be found.